docs/sql-programming-guide.md

```diff
@@ -1253,7 +1253,12 @@ This command builds a new assembly jar that includes Hive. Note that this Hive a
 on all of the worker nodes, as they will need access to the Hive serialization and deserialization libraries
 (SerDes) in order to access data stored in Hive.
 
-Configuration of Hive is done by placing your `hive-site.xml` file in `conf/`.
+Configuration of Hive is done by placing your `hive-site.xml` file in `conf/`. Please note when running 
```

Comment:
the datanucleus jars are added to spark classpath through compute-classpath.sh, but it is not actually added in yarn container. there was JIRA opened to track this SPARK-2624, but the fix is reverted in Dec last year. 

http://mail-archives.apache.org/mod_mbox/incubator-spark-commits/201412.mbox/%3Ceea141c4011644c6b91a8a202a131513@git.apache.org%3E